src/game/board/messageScreen.js

```diff
@@ -0,0 +1,38 @@
+import "./messageScreen.css"
+
+class MessageScreen {
+    constructor(selector) {
+      this.container = document.querySelector(selector);
```

Comment:
Tak też można (i w pewnych sytuacjach może być lepiej), ale w naszym przypadku raczej bym poszedł w stronę konstruktora przyjmującego gotowy węzeł niż selektor - jeśli w kodzie wywołującym MessageScreen wiadomo, jakiego selektora użyć, to mogą go sobie tam wywołać i będą mieli węzeł. Jeśli mają węzeł, to mogą podać wprost. W drugą stronę nie działa to tak łatwo (a nawet, jeśli możesz sobie wyciągnąć selektor z węzła, albo nadać, to zawsze są to dodatkowe operacje do wykonania).